i18n/en-US.yml

```
@@ -52,6 +52,8 @@ actions:
     emailVerificationResent: The email verification message has been resent.
     genericError: "An error was encountered: {err}"
     itineraryExistenceCheckFailed: Error checking whether your selected trip is possible.
+    mustBeLoggedInToSavePlace: Please log in to save locations.
+    placeRemembered: "Your {placeType} has been set to: {address}."
```

`placeType` is either "custom" or "dining" which results in the alert becoming either:
- "Your custom has been set to: Sesame Street."
- "Your dining has been set to: Sesame Street."

Adding something like "place type" would help make this alert more clear
```suggestion
    placeRemembered: "Your {placeType} place type has been set to: {address}."
```

---

Good point! I changed the confirmation popup in https://github.com/opentripplanner/otp-react-redux/pull/992/commits/e1c64536b1f836a7597349d1e54636fe62043861 so it shows the correct place name, and did away with showing the address as there can be other settings for places later.